keyboards/tkc1800/info.json

```diff
@@ -0,0 +1,16 @@
+{
+  "keyboard_name": "TKC1800", 
+  "manufacturer": "The Key Company", 
+  "identifier": "", 
+  "url": "", 
+  "maintainer": "qmk", 
+  "processor": "", 
+  "bootloader": "", 
+  "width": 19.5, 
+  "height": 7, 
+  "layouts": {
+      "LAYOUT": {
```

Comment:
I looked at a couple other keyboards. Would it also be find to just change the info.json to say KEYMAP also?

Reply:
It would work. But we are trying to change everything to LAYOUT to accommodate the new configurator changes. So sooner or later we would have to change it.